Makefile.am

```diff
@@ -157,10 +156,55 @@ mavlink_routerd_SOURCES = \
 	src/common/xtermios.cpp \
 	src/common/xtermios.h
 
+mavlink_routerd_CPPFLAGS = ${AM_CPPFLAGS}
+mavlink_routerd_CPPFLAGS += \
+	-I$(top_srcdir)/src/mavlink-router \
+	-I$(top_builddir)/include/mavlink \
+	-I$(top_builddir)/include/mavlink/ardupilotmega
+
+if RTPS
+bin_PROGRAMS += rtpsd
+rtpsd_SOURCES = \
+	src/common/log.cpp \
+	src/common/log.h \
+	src/common/util.c \
+	src/common/util.h \
+	src/rtps/main.cpp \
+	src/rtps/RtpsTopics.cxx \
```

Comment:
Then it's worse as compatibility will be tied to a specific commit on the px4 tree. The tool to generate these files should probably be extracted in a way that it can be used on both sides of the bridge.  And the tool should probably be improved (if it wasn't already) as I'm seeing a lot of verbose code generated here